config/kafka-instance-types-configuration.yaml

```diff
@@ -44,7 +44,12 @@
 #   - minInSyncReplicas: Minimum number of in sync replicas
 #   - replicationFactor: Replication factor
 #   - supportedAZModes: a list of the supported AZ modes. The possible values are "single", "multi"
-#   - lifespanSeconds: The limit lifespan of the kafka instance in seconds. If not specified then the instance never expires
+#   - [optional] lifespanSeconds: The limit lifespan of the kafka instance in seconds.
+#                                 This attribute is applied at kafka creation time. Updates to this attribute
```

Comment:
Might be good to note what this applies to? to calculate the date of when a Kafka instance expires (marked for deprovision)?

Reply:
:+1: Updated.